projects/bp-gallery/src/graphql/operation.graphql

```diff
@@ -1,71 +1,1332 @@
 # NOTE: this file is auto-generated, edit /projects/bp-graphql/operations instead
-query getAllArchiveTags($sortBy: [String] = ["createdAt:asc"]) { archiveTags(sort: $sortBy) { data { id attributes { name shortDescription showcasePicture { data { id attributes { media { data { attributes { url updatedAt provider } } } } } } } } } }
```

Comment:
Why is this file formatted? [This](https://github.com/hpi-swa-lab/BP2021RH1/blob/adb01f9340bee8a00e8a1785eea7cc09f7ee9faa/projects/bp-gallery/.husky/pretty-quick-ignore) should prevent that.